STYLE.md

```diff
@@ -30,7 +30,8 @@
   fiasco](https://isocpp.org/wiki/faq/ctors#static-init-order-on-first-use) for
   how to best handle this.
 * API-level comments should follow normal Doxygen conventions. Use `@param` to describe
-  parameters, `@return <return-type>` for return values.
+  parameters, `@return <return-type>` for return values. Internal comments for
```

Comment:
OK. We've agreed that comment style for internal docs doesn't matter, but there should be some thought put into making the comment useful (e.g. talking about constraints on arguments etc.) when they are put there. 

@dnoe will own adding proper Doxygen support to the build or stop using Doxygen in his internal comments ;)